src/Juvix/Compiler/Core/Evaluator.hs

```diff
@@ -106,7 +106,7 @@ geval opts herr ctx env0 = eval' env0
           NCtr (Constr _ tag args) ->
             branch n env args tag def bs
           v'
-            | opts ^. evalOptionsNormalize ->
+            | opts ^. evalOptionsNormalize || opts ^. evalOptionsNoFailure ->
```

Comment:
question out of curiosity: What is the purpose of `opts ^. evalOptionsNoFailure` - is it required by a particular backend?

Reply:
Thanks!